apps/snitch_core/test/data/schema/property_test.exs

```diff
@@ -0,0 +1,78 @@
+defmodule Snitch.Data.Schema.PropertyTest do
+  use ExUnit.Case
+  use Snitch.DataCase
+
+  import Ecto.Changeset
+
+  alias Snitch.Data.Schema.Property
+
+  @valid_attrs %{name: "Milk", display_name: "Amul milk"}
+
+  describe "Create property" do
+    test "changeset with valid attributes" do
+      %{valid?: validity} =  Property.create_changeset(%Property{},@valid_attrs)
+      assert validity
+    end
+
+    test "name and display_name cannot be blank" do
+      params = Map.delete(@valid_attrs, :name)
+      cs = %{valid?: validity} = Property.create_changeset(%Property{},params)
+      refute validity
+      assert %{name: ["can't be blank"]} = errors_on(cs)
+
+      params = Map.delete(@valid_attrs, :display_name)
+      cs = %{valid?: validity} = Property.create_changeset(%Property{},params)
+      refute validity
+      assert %{display_name: ["can't be blank"]} = errors_on(cs)
+    end
+
+    test "name must be unique" do
+      cset = Property.create_changeset(%Property{}, @valid_attrs)
```

Comment:
can we use changeset instead of `cset`? it's a better naming convention.

Reply:
@ashish173 Actually cset is used in permission_test.exs , so i thought may be it is used as some convention.